cli/js/repl.ts

```diff
@@ -61,20 +61,29 @@ let lastThrownError: Value = undefined;
 // Returns true if code is consumed (no error/irrecoverable error).
 // Returns false if error is recoverable
 function evaluate(code: string): boolean {
-  const [result, errInfo] = core.evalContext(code);
-  if (!errInfo) {
-    lastEvalResult = result;
-    replLog(result);
-  } else if (errInfo.isCompileError && isRecoverableError(errInfo.thrown)) {
-    // Recoverable compiler error
-    return false; // don't consume code.
+  if (code[0] === ".") {
```

Comment:
`code` should be trimmed before this check.

I hadn't seen this PR at first, leaving this here for reference :p https://github.com/nayeemrmn/deno/commit/23fb0a2f5f475fb7ace4c459daa2e3b5a35e69aa